_episodes/12-cluster.md

```diff
@@ -35,6 +35,8 @@ too big for any one computer.
 
 ## Where are we?
 
+Go ahead and log in to the cluster, using the `ssh` command.
```

Comment:
Please add a code block that does ssh. Let's hard code the machine name as `remote` for now.

Reply:
No objection, can do.

Reply:
I have added a code block, but I unresolved this because you might want another look (once I've pushed the change) -- I've provided a verbose [user@laptop]$ prompt to establish scope, but I can see where this might be extraneous detail.